bin/update

```diff
@@ -13,4 +13,5 @@ end
 require gem_root.join("spec/manageiq/lib/manageiq/environment").to_s
 ManageIQ::Environment.manageiq_plugin_update(gem_root)
 
-ManageIQ::Environment.update_ui
+system('rails db:seed')
```

Comment:
I think this fails because we never run `bin/update`, only `bin/setup` (from `tools/ci/before_install.sh`)

Reply:
I thought it ran bin/setup ... maybe I misread

Reply:
It does. But this is bin/update :)

Reply:
I removed this for now... I agree, my hacky way of setting up the dev database is bad so we might have to make some core changes to how the plugins/rake tasks work but i think this is fine to remove for now.